### PR TITLE
Add VirtualAxis

### PR DESCRIPTION
**Pull request by johanhelsing**
Similar to `VirtualDPad`, but for one axis only.

Closes: #216

### CODE REVIEW

File: src/axislike.rs [hunk 1]
```diff
@@ -334,6 +334,38 @@ impl VirtualDPad {
...
+///
+/// Typically, you don't want to store a [`Axis`] in this type,
+/// even though it can be stored as an [`InputKind`].
+///
+/// Instead, use it directly as [`InputKind::Axis`]!
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub struct VirtualAxis {
+    /// The input that represents the up direction in this virtual DPad
+    pub negative: InputKind,
+    /// The input that represents the down direction in this virtual DPad
```
Comment (johanhelsing) [inline src/axislike.rs:347]: Oops, made some copy paste errors here.

File: src/axislike.rs [hunk 2]
```diff
@@ -334,6 +334,38 @@ impl VirtualDPad {
...
 
+/// A virtual Axis that you can get an [`Axis`] from
+///
+/// Typically, you don't want to store a [`Axis`] in this type,
+/// even though it can be stored as an [`InputKind`].
+///
+/// Instead, use it directly as [`InputKind::Axis`]!
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub struct VirtualAxis {
+    /// The input that represents the up direction in this virtual DPad
```
Comment (alice-i-cecile) [inline src/axislike.rs:345]:
```suggestion
    /// The input that represents the down direction in this virtual axis
```

File: src/axislike.rs (hunk 1 above)
Comment (alice-i-cecile) [inline src/axislike.rs:347]:
```suggestion
    /// The input that represents the up direction in this virtual axis
```

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Solid work! A minor doc suggestion, and this needs to be added to the change log.

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**